go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/openshift/certman-operator
 
-go 1.23.6
+go 1.24.5
```

Comment:
As discussed, I think we should stick to 4.19 equivalent deps for now with Go 1.23 and Kube 1.32 and controller-runtime v0.20.0. Other deps we can update in next iteration with 4.20 update